balloon/src/main/java/com/skydoves/balloon/Balloon.kt

```diff
@@ -1226,7 +1226,7 @@ class Balloon(
     val spaces = rootView.paddingLeft + rootView.paddingRight + if (builder.iconDrawable != null) {
       builder.iconWidth + builder.iconSpace
     } else 0 + builder.marginRight + builder.marginLeft + (builder.arrowSize * 2)
-    val maxTextWidth = displayWidth - spaces
+    val maxTextWidth = builder.maxWidth - spaces
```

Comment:
I found a bug with `maxTextWidth` in `getMeasuredTextWidth` that when we had longer texts we would get this behavior 
![image](https://user-images.githubusercontent.com/36191409/128675148-47b0bd0b-057a-40a2-8b50-b5d4be356c5b.png)
when the expected behavior should be like the following 
![image](https://user-images.githubusercontent.com/36191409/128675264-96911dca-6c3e-4115-b759-f9e70c6edcae.png)